src/frontend/apps/lti_site/components/LocalizedTimedTextTrackUpload/index.spec.tsx

```diff
@@ -50,15 +49,24 @@ describe('<LocalizedTimedTextTrackUpload />', () => {
     });
   });
 
-  it('renders the component without any uploaded timed text track', () => {
+  afterEach(() => {
+    fetchMock.restore();
+  });
+
+  it('renders the component without any uploaded timed text track', async () => {
+    fetchMock.mock(
```

Comment:
Same as before, if the same mock is in all the tests, considere to add it in the beforeEach methods.

Reply:
Same as before, I mock a 500 in the last test